Sources/SwiftLibraryPluginProvider/LibraryPluginProvider.swift

```diff
@@ -137,7 +139,7 @@ private func _loadLibrary(_ path: String) throws -> UnsafeMutableRawPointer {
 #else
 private func _loadLibrary(_ path: String) throws -> UnsafeMutableRawPointer {
   guard let dlHandle = dlopen(path, RTLD_LAZY | RTLD_LOCAL) else {
-    throw LibraryPluginError(message: "loader error: \(String(cString: dlerror()))")
+    throw LibraryPluginError(message: "loader error: \(String(cString: dlerror()!))")
```

Comment:
Does this work with glibc? I thought that it was annotated with non null in some cases (at least I ran into one with foundation)

Reply:
Yep, worked fine in Fedora 40 x86_64, where I tested it before submitting, which uses a recent glibc 2.39-17 from earlier this year. Bionic simply added the `_Nullable` annotation to the output in the last year, so it _has_ to be unwrapped for Android here, but unless glibc somehow can guarantee this is `_Nonnull`, this force unwrap should work for both.